src/lib.rs

```diff
@@ -1189,6 +1183,15 @@ pub unsafe fn string_from_c_str(c_str: *const c_char) -> String {
     String::from_utf8_lossy(CStr::from_ptr(c_str).to_bytes()).into_owned()
 }
 
+/// Like `string_from_c_str`, but handles null pointers correctly
+pub unsafe fn string_from_nullable_c_str(c_str: *const c_char) -> Option<String> {
```

Comment:
Perhaps remove `pub`?

Reply:
I made this `pub` as the original `string_from_c_str` is `pub` as well. I think either both or none should be `pub`.

Reply:
Ah, OK.